stores/slack/README.md

```diff
@@ -106,7 +106,7 @@ type channel
     # channels have a parent workspace
     define parent_workspace: [workspace]
     # users can be writers on channels
-    define writer: [user]
+    define writer: [user,workspace#member]
```

Comment:
Do we also need to update commenter to be [user,workspace#member]?

Reply:
We don't need that for the tuples. but yeah that's a good idea, will add it

Reply:
Updated